projects/npm-tools/packages/npm-scripts/src/jest/mocks/AUI.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* eslint-env jest */
+
+const AUI = () => ({
```

Comment:
👀

Reply:
👀

Reply:
This is unnecessary. I tried to see usages of AUI in `data-engine-taglib` but I didn't found

Reply:
Do they still use it? did you get to test without it?

Reply:
> Do they still use it? 

Nope

> did you get to test without it?

Yes, it's working without AUI.
Methodology:
1. Checkout master
2. Remove AUI in `jest-setup.config.js`
3. `yarn test`
4. See it working without any fail